changelog/unreleased/pull-2655

```diff
@@ -0,0 +1,5 @@
+Bugfix: Addition of --min-packsize and --file-read-concurrency
+
+In order to tune restic for large installs we need to be able to configure --min-packsize and --file-read-concurrency. I the current release these numbers are hard coded.
```

Comment:
This is written like a feature request. Also, "the current release" is the one that would incorporate this patch.

Reply:
modified